src-java/kilda-pce/src/main/java/org/openkilda/pce/model/PathWeight.java

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.pce.model;
+
+import static com.google.common.primitives.Longs.asList;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Path weight.
+ */
+public class PathWeight implements Comparable<PathWeight> {
+
+    private List<Long> params;
```

Comment:
I guess we need so generic param because PathWeight will be used with different strategies. And different strategies can have different param names.